pkg/role_binding/role_binding.go

```diff
@@ -62,6 +64,35 @@ func CreateRoleBinding(accountID string, roleID string, roleType string, resourc
 	return rb, nil
 }
 
+func CreateAccountGroupRoleBinding(accountGroupID string, roleID string, roleType string, resourceID *string) (*v1.RoleBinding, error) {
+	newRoleBinding := v1.NewRoleBinding().AccountGroupID(accountGroupID).RoleID(roleID).Type(roleType)
+	if roleType == SubscriptionRoleBinding {
+		newRoleBinding.SubscriptionID(*resourceID)
+		newRoleBinding.ManagedBy(ManagedByUser)
+	}
+	if roleType == OrganizationRoleBinding {
+		newRoleBinding.OrganizationID(*resourceID)
+	}
+	rb, err := newRoleBinding.Build()
+	if err != nil {
+		return nil, fmt.Errorf("can't create account group role binding: %v", err)
+	}
+	return rb, nil
+}
+
+func AddAccountGroupRoleBinding(accountGroupID string, roleID string, roleType string, resourceID *string, conn *sdk.Connection) (*v1.RoleBinding, error) {
```

Comment:
Is this being used somewhere?

Reply:
No, this (and CreateAccountGroupRoleBinding) should be removed. I created them from existing patterns but YAGNI.